### ISSUE TITLE
Loadable breaks tests if used with SSR

### ISSUE THREAD

**Issue by wagnerjsilva** [smooth-code/loadable-components#396]
## 🐛 Bug Report

Loadable components are breaking tests on Razzle > 3.0 with error: 
TypeError: template.ast is not a function

## To Reproduce

 npm run test -- --coverage

Package json: 

`

{
  "name": "my-razzle-app",
  "version": "3.0.0",
  "license": "MIT",
  "scripts": {
    "start": "razzle start",
    "start_uat": "RAZZLE_ENV=uat razzle start",
    "start_production": "RAZZLE_ENV=production razzle start",
    "build": "razzle build",
    "test": "razzle test --env=jsdom",
    "test:debug": "node --inspect node_modules/.bin/jest --watch --runInBand",
    "test:coverage": "jest --verbose --coverage",
    "start:prod": "NODE_ENV=production node build/server.js"
  },
  "dependencies": {
    "@babel/runtime": "^7.1.5",
    "@loadable/component": "^5.10.2",
    "@loadable/server": "^5.10.2",
    "@sentry/browser": "^4.6.6",
    "axios": "^0.18.0",
    "compress": "^0.99.0",
    "enzyme": "^3.10.0",
    "enzyme-adapter-react-16": "^1.14.0",
    "express": "4.16.3",
    "identity-obj-proxy": "^3.0.0",
    "intersection-observer": "^0.7.0",
    "jest-css-modules": "^1.1.0",
    "mobile-detect": "^1.4.3",
    "moment": "^2.24.0",
    "prop-types": "^15.7.2",
    "radium": "^0.24.1",
    "razzle": "^3.0.0",
    "react": "^16.0.0",
    "react-cookie": "^3.1.2",
    "react-dom": "^16.0.0",
    "react-google-charts": "^3.0.13",
    "react-helmet": "^5.2.1",
    "react-redux": "^5.1.1",
    "react-router-dom": "4.3.1",
    "react-select": "^2.4.3",
    "react-slick": "^0.23.2",
    "react-test-renderer": "^16.8.6",
    "react-transition-group": "^2.9.0",
    "redux": "^4.0.1",
    "redux-thunk": "^2.3.0",
    "scroll-to-element": "^2.0.3",
    "slick-carousel": "^1.8.1",
    "universal-cookie-express": "^3.1.0"
  },
  "jest": {
    "moduleNameMapper": {
      "\\.(css|less)$": "identity-obj-proxy"
    }
  },
  "devDependencies": {
    "@loadable/babel-plugin": "^5.10.0",
    "@loadable/webpack-plugin": "^5.7.1",
    "check-prop-types": "^1.1.2"
  }
}

`

**Comment by wagnerjsilva** [comment]
I might be completely wrong here, but it seems that when I run the tests it needs to have template.ast: 

https://babeljs.io/docs/en/babel-template

Available to: 

@loadable/babel-plugin

However, I can't see that dependency being included.

**Comment by neoziro** [comment]
You probably have to install it in devDependencies.

**Comment by wagnerjsilva** [comment]
would you be able to advise which devDependencies are missing here: 

},
"devDependencies": {
"@loadable/babel-plugin": "^5.10.0",
"@loadable/webpack-plugin": "^5.7.1",
"check-prop-types": "^1.1.2"
}

**Comment by wagnerjsilva** [comment]
Apologies, I see what you mean now. Will try it soon.

**Comment by wagnerjsilva** [comment]
npm install --save-dev @babel/template resolved the issue.

Thank you!

**Comment by wagnerjsilva** [comment]
Here is a full trace from one of the errors: 

`  ● Test suite failed to run

    TypeError: template.ast is not a function

      at isReadyProperty (node_modules/@loadable/babel-plugin/lib/properties/isReady.js:10:31)
      at properties.map.init (node_modules/@loadable/babel-plugin/lib/index.js:39:52)
          at Array.map (<anonymous>)
      at loadablePlugin (node_modules/@loadable/babel-plugin/lib/index.js:39:40)
      at Function.memoisePluginContainer (node_modules/babel-core/lib/transformation/file/options/option-manager.js:113:13)
      at Function.normalisePlugin (node_modules/babel-core/lib/transformation/file/options/option-manager.js:146:32)
      at node_modules/babel-core/lib/transformation/file/options/option-manager.js:184:30
          at Array.map (<anonymous>)
      at Function.normalisePlugins (node_modules/babel-core/lib/transformation/file/options/option-manager.js:158:20)
      at OptionManager.mergeOptions (node_modules/babel-core/lib/transformation/file/options/option-manager.js:234:36)`

**Comment by wagnerjsilva** [comment]
ok, so it looks like I finally got to the bottom of the issue.

I had two problems:

1) An old .babelrc file was being used 
2) After removing the file it became aparent razzle was using an older version of Jest. So I've installed the latest version of it, and it worked.

This thread helped me: 

https://github.com/FormidableLabs/spectacle/issues/522#issuecomment-423019983

Hope this helps some poor soul out there :-)
